src/module_ncio.f90

```diff
@@ -170,31 +189,27 @@ integer function get_ndim(dset, dimname)
        endif
     enddo
   end function get_ndim
-
-  !>
+  !> get Variable object given name
   !! 
   !! @param dset 
   !! @param varname 
   !! 
   !! @return 
   function get_var(dset, varname) result (var)
-    ! get Variable object given name
     type(Dataset) :: dset
     type(Variable) :: var
     character(len=*) :: varname
     integer nvar
     nvar = get_nvar(dset, varname)
     var = dset%variables(nvar)
   end function get_var
-
-  !>
+  !> returns .true. is varname exists in dset, otherwise .false.
```

Comment:
Use the @return flag for this info, put it at the bottom as it is here, but with the information next to it, like this:

`!! @return .true. is varname exists in dset, otherwise .false.
`